### PR TITLE
Add official release workflows

### PR DESCRIPTION
**Pull request by alextaing**
`version_update` and `sync_develop_and_main` workflows were added.

J=SLAP-2159
TEST=manual

Forked this repo, made a release branch and merged into main.  2 PRs were made: one updating for package version and one for merging main into develop.

### CODE REVIEW

File: .github/workflows/sync_develop_and_main.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Create PR from main to develop
+
+on:
+  push:
+    branches: [main, master]
```
Comment (nmanu1) [inline .github/workflows/sync_develop_and_main.yml:5]: since we don't have a `master` branch in this repo, we can just run on pushes to `main`

**Review by nmanu1 — COMMENTED** [review]
can this PR be changed to merge into `develop` instead of `main`?

File: .github/workflows/sync_develop_and_main.yml (hunk 1 above)
Comment (alextaing) [reply .github/workflows/sync_develop_and_main.yml:5]: Sure!  I was considering that.  I assume I should do that for headless as well.

**Comment by alextaing** [comment]
tested manually on a forked repo!

**Review by nmanu1 — APPROVED** [review]

**Pull request merged**